templates/cases/inbox_messages.haml

```diff
@@ -24,6 +24,8 @@
         %form.clearfix
           .btn-group.pull-away
             // TODO figure out why form is double-submitted when this is moved to ng-submit on the form. jQuery conflict?
+            %button.btn.btn-default{ type:"button", ng-click:"onCaseWithoutMessage()", ng-if:'folder == "inbox"'}
```

Comment:
We need to disable this for now on upartners - so maybe a setting like `SITE_ALLOW_CASE_WITHOUT_MESSAGE`